### PR TITLE
Added className to Calendar component

### PR DESCRIPTION
**Pull request by lafe**
#### Pull request checklist

- [X] Addresses an existing issue: Implements change discussed in #4514 
- [X] Include a change request file using `$ npm run change`

#### Description of changes

This change adds an optional "`className`" property to the `ICalendarProps` interface. If a class is specified, it will be added to the root element of the `Calendar` component.

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Calendar/Calendar.types.ts [hunk 1]
```diff
@@ -16,6 +16,9 @@ export interface ICalendarProps extends React.Props<Calendar> {
    */
   componentRef?: (component: ICalendar | null) => void;
 
+  /** Optional class name to add to the root element. */
```
Comment (manishgarg1) [inline packages/office-ui-fabric-react/src/components/Calendar/Calendar.types.ts]: >/** [](start = 2, length = 3)

Please make the comment to be multi line. Like 

/**
 * blah blah blah...
 */

**Comment by manishgarg1** [comment]
Otherwise looks good to me. @dzearing any concerns?

**Comment by lafe** [comment]
Fixed comment (however, I copied that comment from another component in the Office UI React Framework... :smile:)

File: packages/office-ui-fabric-react/src/components/Calendar/Calendar.test.tsx [hunk 2]
```diff
@@ -173,6 +173,7 @@ describe('Calendar', () => {
           dateRangeType={ DateRangeType.Week }
           autoNavigateOnSelection={ true }
           onSelectDate={ onSelectDate() }
+          className="CalendarTestClass"
```
Comment (jordandrako) [inline packages/office-ui-fabric-react/src/components/Calendar/Calendar.test.tsx]: Travis is failing due to the double quote marks. Change all `"` to `'` for it to pass.

**Comment by cliffkoh** [comment]
Hello @lafe, could you resolve the merge conflict and let's see if CI passes this time so we can get this merged in? :)

**Pull request closed without merging**